examples/java-matter-controller/java/src/com/matter/controller/commands/common/RealResult.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+// RealResult encapsulate the boolean result, and error string if the result is false.
```

Comment:
same: Doc-comment to the class. Should look like:

```
/**
  * Implements a Result where an error string is associated with its failure state.
  *
  * A `Result` is just a booean of true/false that is exposed via `getResult`. This class
  * will contain either a `true` value for `Success` or a `false` value in which case
  * the failure will also have an error string explaining the reason of the failure associated with it.
  */
```
public class RealResult implements Result {
//...
```

Reply:
updated, thanks